.travis.yml

```diff
@@ -1,11 +1,12 @@
 language: python
 python:
   - "2.7"
-  - "3.4"
+  - "3.4"  # When support for 3.4 is removed unpin the PyYAML version below.
   - "3.6"
 # command to install dependencies
 install:
-  - pip install PyYAML argparse rosdep vcstools catkin-pkg python-dateutil setuptools
+  - pip install PyYAML==5.2 # Forcing PyYAML 5.2 while we retain Python 3.4 support PyYAML 5.3 and higher does not support python 3.4
```

Comment:
The `PyYAML` version shouldn't be pinned for Python version != 3.4. See dirk-thomas/vcstool#117 for an example.